rest-server/bin/harness-env

```diff
@@ -8,9 +8,7 @@ export REST_SERVER_HOST=${REST_SERVER_HOST:-localhost}
 export REST_SERVER_PORT=${REST_SERVER_PORT:-9090}
 
 # MongoDB setup
-# todo: allow for multiple servers in cluster
-export MONGO_HOST=${MONGO_HOST:-localhost}
-export MONGO_PORT=${MONGO_PORT:-27017}
+export MONGO_URI=${MONGO_URI:-mongodb://localhost:27017}
 
```

Comment:
ideally this creates MONGO_URI from MONGO_HOST and MONGO_PORT because these are passed into the container in the docker-composes we maintain along with k8s configs

Also if we want to pass in URIs we should probably do that everywhere and fix this only once in docker-compose and k8s